turbo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**", "!.next/cache/**"]
+    }
```

Comment:
Can you add `"dependsOn": ["^build"]` just like in https://turbo.build/repo/docs/core-concepts/caching#configuring-cache-outputs

Reply:
That interessing